account_financial_report/models/account_move_line.py

```diff
@@ -15,25 +16,27 @@ class AccountMoveLine(models.Model):
     @api.depends("analytic_distribution")
     def _compute_analytic_account_ids(self):
         # Prefetch all involved analytic accounts
-        with_distribution = self.filtered("analytic_distribution")
-        batch_by_analytic_account = defaultdict(list)
-        for record in with_distribution:
-            for account_id in map(int, record.analytic_distribution):
-                batch_by_analytic_account[account_id].append(record.id)
+        batch_by_analytic_account = defaultdict(lambda: self.env["account.move.line"])
+        for record in self.filtered("analytic_distribution"):
+            # NB: ``analytic_distribution`` is a JSON field where keys are:
+            # - 'project.id' if only the project is set
```

Comment:
Here project has nothing to do. It's the analytic account. I don't understand either what is the tag you mention.

Reply:
The columns in the widget are labelled "Project" and "Tag" (or "Departments", depending on the installed modules).
I'm just using their labels to reference them, but I know they are in fact analytic accounts, see the rest of the comment.

Reply:
Where do you see that nomenclature? What I see in the widget are analytic plans:

![imagen](https://github.com/user-attachments/assets/0f49e009-358d-469e-9504-320e858c51e8)

It's true that the standard demo data has a plan called "Projects", but it can be any name.

Reply:
I will update the comment and the commit message